src/pageServices/contentScripts/highlight.js

```diff
@@ -286,7 +289,13 @@ export const highlightOnPage = () => {
             locatorType === "CSS selector"
               ? findBySelector(locator.output, element_id)
               : findByXpath(locator.output, element_id);
+          if (!jdnHash) {
```

Comment:
What do you think about moving this check and searching new hash before `let node = findByHash(jdnHash);`?